spec/rails_helper.rb

```diff
@@ -38,6 +38,13 @@
   exit 1
 end
 RSpec.configure do |config|
+  Shoulda::Matchers.configure do |config|
```

Comment:
I'd suggest you to move this code to `spec/support/shoulda.rb`

Reply:
done :+1: